### PR TITLE
Register member

### PR DESCRIPTION
**Pull request by DinolanNaidoo**

### CODE REVIEW

File: skietbaan_fe/src/components/NavbarMenu.js [hunk 1]
```diff
@@ -98,6 +98,39 @@ class NavbarMenu extends Component {
 		}
 	}
 
+	isProfilev2() {
+		if (window.location.pathname.endsWith("/profile")) {
+			return <img src={require('../components/navbar-icons/profile-red.png')}
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/NavbarMenu.js]: extract this to a variable by importing it in the beginning 
eg `import profileRed from '../components/navbar-icons/profile-red.png'`
then `src={profileRed}`

Comment (FredJEhlers) [reply skietbaan_fe/src/components/NavbarMenu.js]: Do it for all please

File: skietbaan_fe/src/bootstrap/NavbarMenuStyle.css [hunk 2]
```diff
@@ -13,12 +15,90 @@
...
+
+  .second-row-navbar{
+    height: 68px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    width: 100%;
+    display: none;
+  }
+
+  .columns-v2{
```
[the remark this answers is not in the document]
Comment (DinolanNaidoo) [reply skietbaan_fe/src/bootstrap/NavbarMenuStyle.css]: The reason i did that was for the different views i.e. desktop and mobile. so columns-v2 hides on mobile and shows and desktop. And if i take it out it doesn't display properly.

File: skietbaan_fe/src/components/NavbarMenu.js (hunk 1 above)
Comment (DinolanNaidoo) [reply skietbaan_fe/src/components/NavbarMenu.js]: made changes for all

**Pull request merged**